rasa/shared/core/generator.py

```diff
@@ -159,23 +163,31 @@ def init_copy(self) -> "TrackerWithCachedStates":
             self.is_rule_tracker,
         )
 
+    @contextmanager
+    def _skip_states_manager(self) -> bool:
+        self._skip_states = True
+        try:
+            yield self._skip_states
```

Comment:
it doesn't seem that this variable is read, no need to yield it I guess?

Reply:
Good point, does that mean I should only keep `yield` without the variable?